cves/CVE-2013-0894.yml

```diff
@@ -90,20 +95,29 @@ fixes_vcc_instructions: |
   find interesting in there.
 fixes:
 - commit: 2c16bf2de07c68513072bf3cc96401d2c6291a3e
-  note: ''
+  note: 'This fixes potentail divisions by zero and out anf array accessees.'
 - commit: 11dcecfcca0eca1a571792c4fa3c21fb2cfddddc
-  note: ''
+  note: 'The Value is used to calculate output LSP curve and a division by zero and out of array accesses would occur.'
```

Comment:
What is an LSP curve? If it's part of the problem, then you need to mention it in the description.